routes/restaurantOwners.js

```diff
@@ -0,0 +1,22 @@
+var express = require('express');
+var router = express.Router();
+
+var controller = require('../controllers/restaurantOwners.controller');
+
+//router.get('/', function (req, res, next) {
+  //  res.redirect('/restaurantowners');
+//});
+
+router.get('/', controller.showHomePage);
+
+router.get(
+    '/:uname',
+    controller.showHomePage
+);
+
+router.get(
+    '/:uname/:rname',
+    controller.showRestaurant
+);
```

Comment:
Think once we implement login we don't need to take in uname as a parameter; just leaving here first!